ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/router-isis.yml

```diff
@@ -18,10 +18,19 @@ router_isis:
   timers:
     local_convergence:
       protected_prefixes: true
+      delay: 15000
+    csnp:
+      generation:
+        interval: 10
```

Comment:
Gave priority to host1 for adding tests. Checked that we have no keys missing in coverage.